_docs/README.md

```diff
@@ -66,7 +76,7 @@ boilerplate \
   --non-interactive \
   --var Title="Boilerplate Example" \
   --var WelcomeText="Welcome!" \
-  --var ShowLogo="y"
+  --var ShowLogo="true"
```

Comment:
Not strictly necessary here since it's one word, but a good practice to follow in general.